cookieDriver.js

```diff
@@ -0,0 +1,290 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import {get, getAll, remove, set} from 'tiny-cookie';
+
+export default {
+    _driver: 'cookieWrapper',
+    _initStorage,
+    iterate,
+    getItem,
+    setItem,
+    removeItem,
+    clear,
+    length,
+    key,
+    keys,
+    dropInstance
+};
+
+function _initStorage(options) {
+  const dbInfo = {};
+  if(options) {
+    Object.assign(dbInfo, options);
+  }
+  dbInfo.keyPrefix = _getKeyPrefix(options, this._defaultConfig);
+  // FIXME: support `serializer`?
+  //dbInfo.serializer = serializer;
+  this._dbInfo = dbInfo;
+  return Promise.resolve();
```

Comment:
If the intention here is to pave the way for a possible async operation in `_initStorage`, would making this an `async function` and then not doing this `Promise.resolve()` business be more clear?